test/test_core.py

```diff
@@ -6542,6 +6559,8 @@ def test_neon_wasm_simd(self):
   @no_safe_heap('has unaligned 64-bit operations in wasm')
   @no_ubsan('test contains UB')
   def test_sse1(self):
+    if self.is_wasm64():
+      self.skipTest('https://github.com/llvm/llvm-project/issues/57577')
```

Comment:
why the explicit self.skipTest in some places and @no_wasm64 in others?

Reply:
Hmm.. I think that must have been the result of some intermediate step.  Fixed.